hook.php

```diff
@@ -74,7 +74,7 @@ function plugin_formcreator_addDefaultJoin($itemtype, $ref_table, &$already_link
    switch ($itemtype) {
       case "PluginFormcreatorIssue" :
          $join = Search::addDefaultJoin("Ticket", "glpi_tickets", $already_link_tables);
-         $join = str_replace('`glpi_tickets`', '`glpi_plugin_formcreator_issues`', $join);
+         $join = str_replace('`glpi_tickets`.`id`', '`glpi_plugin_formcreator_issues`.`original_id`', $join);
```

Comment:
on this part, i think we don't have anymore '`glpi_tickets`.`id`' due to the previous replace